.github/project.yml

```diff
@@ -1,4 +1,3 @@
 release:
-  current-version: 0.0.1
-  next-version: 1.0.0-SNAPSHOT
-
+  current-version: 1.0.0
+  next-version: 1.0.1-SNAPSHOT
```

Comment:
To avoid bumping the development version, I'd recommend using 999-SNAPSHOT for the main branch

```suggestion
  next-version: 999-SNAPSHOT
```

Reply:
okay, updated.